apps/dashboard/src/@/components/ui/ScrollShadow/ScrollShadow.tsx

```diff
@@ -94,36 +94,36 @@ export function ScrollShadow(props: {
       }
     >
       <div
-        className={`${styles.scrollShadowTop} ${styles.scrollShadowY}`}
+        className={`${styles.scrollShadowTop}${styles.scrollShadowY}`}
         ref={shadowTopEl}
         style={{
           opacity: "0",
           display: props.disableTopShadow ? "none" : "block",
         }}
       />
       <div
-        className={`${styles.scrollShadowBottom} ${styles.scrollShadowY}`}
+        className={`${styles.scrollShadowBottom}${styles.scrollShadowY}`}
         ref={shadowBottomEl}
         style={{
           opacity: "0",
         }}
       />
       <div
-        className={`${styles.scrollShadowLeft} ${styles.scrollShadowX}`}
+        className={`${styles.scrollShadowLeft}${styles.scrollShadowX}`}
```

Comment:
Wrong formatting here by biome - breaks the layout

Reply:
Same issue in this file 3 more times

Reply:
fixed by using `cn` for this an any other case of dynamic classNames using template strings